integration/utils/contLoadGenerator.go

```diff
@@ -89,21 +102,56 @@ func NewContLoadGenerator(
 		return nil, err
 	}
 
+	// check and cap params for const-exec mode
+	if loadType == ConstExecCostLoadType {
+		if maxConstExecTxSizeInByte > flow.DefaultMaxTransactionByteSize {
+			log.Warn().Msg(
```

Comment:
just return error instead of fudging the value?  same below

Reply:
done